### PR TITLE
Hotfix(new features)

### PR DESCRIPTION
**Pull request by SpeedyD**
Forgot for a moment my testing environment is different from the dev environment, and that I shouldn't make last minute text edits.

Hotfix for 500 errors caused by BOTH #948 and #949 - my bad, sorry.

### CODE REVIEW

**Review by AW0005 — APPROVED** [review]

**Comment by SpeedyD** [comment]
wait. Dangit, hang on.

**Review by AW0005 — APPROVED** [review]

**Comment by SpeedyD** [comment]
Okay __now__ I'm done. Oh my god I missed so many tiny things, I'm so sorry.

**Review by AW0005 — APPROVED** [review]

**Comment by SpeedyD** [comment]
That one was just aesthetics. Forgot a space. Not so much necessary for this hotfix.

**Review by Ne-wt — APPROVED** [review]

File: resources/views/admin/items/create_edit_item.blade.php [hunk 1]
```diff
@@ -102,7 +102,7 @@
         @endif
         <div class="col-md form-group">
             {!! Form::checkbox('is_deletable', 1, $item->id ? $item->is_deletable : 1, ['class' => 'form-check-input', 'data-toggle' => 'toggle']) !!}
-            {!! Form::label('is_deletable', 'Can Be Deleted', ['class' => 'form-check-label ml-3']) !!} {!! add_help('If this is off, users will not be able to delete the item from their or their character's inventory.') !!}
+            {!! Form::label('is_deletable', 'Can Be Deleted', ['class' => 'form-check-label ml-3']) !!} {!! add_help('If this is off, users will not be able to delete the item from the user or character inventory.') !!}
```
[the remark this answers is not in the document]
Comment (SpeedyD) [reply resources/views/admin/items/create_edit_item.blade.php]: 👍 Done and done

**Review by itinerare — APPROVED** [review]

**Pull request merged**